### PR TITLE
Police gear additions

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
```SUMMARY: Content "Police gear additions"```

#### Purpose of change
Adding and modifying various items and the riot control officer profession for the sake of realism.

#### Describe the solution

**Added riot armor**

There is only riot helmet in the game currently, however getting riot armor would be one of the first ideas coming to mind to get protection. Riot armor is mid-low tier armor spawning on cops and in police stations. It is light and easy to fight in, but it is not a ballistic vest, just like real life riot armors.

**Added police duty belt**

Police duty belt is a leather belt used by police officers, has pouches for storage and a holder for a baton. Currently there is no way to wear batons and tonfas on the belt, they have to be fished out of the backpack every time the player wants to change weapons. It spawns on cops and in police stations.

**Added belt_clip to tonfa-like weapons**

These include tonfa, wooden tonfa, tactical tonfa and PR-24 baton. The usual storage for these weapons is that the officer wears them on his belt. All of them can be attached to the police duty belt (the tactical tonfa was too big even for the firefighter belt).

**Modified Riot Control Officer**

Added police duty belt and swapped MBR vest with riot armor. Per my understanding MBR vest is a ballistic vest favored by the military and SWAT or other special forces. Now the profession is more in line with reality by having a riot armor instead of a ballistic vest.

#### Additional context
Riot armor:
https://www.securityprousa.com/collections/riot-gear/products/damascus-gear-upper-body-and-shoulder-protector

Police duty belt:
https://www.galls.com/uncle-mikes-sentinel-duty-gear-kit?PMWTNO=000000000002120&PMSRCH=

### CODE REVIEW

File: data/json/items/armor.json [hunk 1]
```diff
@@ -627,6 +627,28 @@
     "material_thickness": 5,
     "flags": [ "VARSIZE", "OUTER", "FANCY" ]
   },
+  {
+    "id": "armor_riot",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": "riot armor",
+    "description": "Black armor used by riot police officers.  The padded layers and hard plastic shoulder guards offer good protection without much encumberance.  The word POLICE is emblazoned across the front.",
```
Comment (Night-Pryanik) [inline data/json/items/armor.json]: Encumbrance.

File: data/json/items/armor.json [hunk 2]
```diff
@@ -627,6 +627,28 @@
...
+	"looks_like": "kevlar",
```
Comment (Night-Pryanik) [inline data/json/items/armor.json]: Tabs instead of spaces.

File: data/json/items/armor.json [hunk 3]
```diff
@@ -627,6 +627,28 @@
...
+    "weight": 3000,
```
Comment (Night-Pryanik) [inline data/json/items/armor.json]: It's only three kilograms?

File: data/json/items/armor.json [hunk 4]
```diff
@@ -7170,6 +7192,33 @@
     "material_thickness": 1,
     "flags": [ "OVERSIZE", "BELTED", "RESTRICT_HANDS", "WATER_FRIENDLY" ]
   },
+  {
+    "id": "police_belt",
+    "type": "ARMOR",
+    "name": "police duty belt",
+    "description": "Black leather belt used by police officers.  It has several pouches and a holder for a baton.",
+	"looks_like": "fireman_belt",
```
Comment (Night-Pryanik) [inline data/json/items/armor.json]: Tabs.

File: data/json/items/armor.json [hunk 5]
```diff
@@ -7170,6 +7192,33 @@
...
+    "material": [ "leather", "steel" ],
+    "symbol": "[",
+    "color": "brown",
+    "covers": [ "TORSO" ],
+    "encumbrance": 4,
+    "storage": 8,
+    "use_action": {
+      "type": "holster",
+      "holster_prompt": "Attach what to holder?",
+      "holster_msg": "You attach your %s to your %s",
```
Comment (Night-Pryanik) [inline data/json/items/armor.json]: Period in the end.

File: data/json/items/armor.json (hunk 1 above)
Comment (echomirage1) [reply data/json/items/armor.json]: Corrected.

File: data/json/items/armor.json (hunk 2 above)
Comment (echomirage1) [reply data/json/items/armor.json]: Corrected.

File: data/json/items/armor.json (hunk 3 above)
Comment (echomirage1) [reply data/json/items/armor.json]: I've changed it to 4 kilograms, but according to the site below the full suit with arm and leg guards is 6,5 kilograms.

https://www.armorusa.com/product/riogard-riot-gear/

File: data/json/items/armor.json (hunk 4 above)
Comment (echomirage1) [reply data/json/items/armor.json]: Corrected.

File: data/json/items/armor.json (hunk 5 above)
Comment (echomirage1) [reply data/json/items/armor.json]: Corrected. It is a copy-paste from fireman_belt so it would need change there also.

**Pull request merged**